src/makeRobotAccountant.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  const CORRECT_COUNT_WORKINGS = 3;
+  let callCount = 1;
+
+  return (firstNum) => (secondNum) => {
+    const isEven = callCount % 2 === 0;
+    const isBroken = callCount > CORRECT_COUNT_WORKINGS && isEven;
```

Comment:
```suggestion
  const FINE_CALLS = 3;
  let callCount = 1;

  return (firstNum) => (secondNum) => {
    const isEvenCall = callCount % 2 === 0;
    const isBroken = callCount > FINE_CALLS && isEvenCall;
```
Consider better naming.